command/src/channel.rs

```diff
@@ -556,7 +547,7 @@ mod tests {
             Channel<Serializable, Serializable>,
             Channel<Serializable, Serializable>,
         ) = Channel::generate(1000, 10000).expect("Could not create nonblocking channel");
-        writing_channel.blocking();
+        writing_channel.blocking().unwrap();
```

Comment:
To me, this should be replaced by an `expect`

Reply:
done